src/doc-utils.js

```diff
@@ -21,6 +21,10 @@ function traverseDoc(doc, onEnter, onExit) {
       if (doc.flatContents) {
         traverseDocRec(doc.flatContents);
       }
+    } else if (doc.type === "group" && doc.expandedStates) {
+      if (shouldTraverseConditionalGroups) {
+        doc.expandedStates.forEach(traverseDocRec);
+      }
```

Comment:
Just to clarify, as my above comment is wrong, I see that it will enter only this branch if it's a conditional group. But `expandedStates` are all of the conditions, including the first one. `contents` happens to point to the first one for convenience.